graphein/molecule/edges/atomic.py

```diff
@@ -0,0 +1,28 @@
+"""Functions for computing atomic structure of proteins."""
+import logging
+
+# %%
+# Graphein
+# Author: Arian Jamasb <arian@jamasb.io>
+# License: MIT
+# Project Website: https://github.com/a-r-j/graphein
+# Code Repository: https://github.com/a-r-j/graphein
+from typing import Any, Dict
+
+import networkx as nx
+import numpy as np
+import pandas as pd
+
+
+log = logging.getLogger(__name__)
+
+def add_atom_bonds(G: nx.Graph) -> nx.Graph:
+    for bond in G.graph["rdmol"].GetBonds():
+        n1, n2 = bond.GetBeginAtomIdx(), bond.GetEndAtomIdx()
+        if G.has_edge(n1, n2):
+            G.edges[n1, n2]["kind"].add("bond")
+        else:
+            G.add_edge(n1, n2, kind={"bond"})
```

Comment:
I think we should have an option to distinguish bond order (single/double/triple etc)

Reply:
Uh you are right, do we make it as edge feature or mark here?

Reply:
I think we can do it as the kind. E.g `kind={"single", "bond"}` `or kind={"single_bond"}`. I think the first one may be easier to work with (and is what I did for [atomic protein graphs](https://github.com/a-r-j/graphein/blob/ac66a1d3aee8116d4b155ae8d7bd80235d3f247f/graphein/protein/edges/atomic.py#L177))